src/MoBi.Presentation/MenusAndBars/ContextMenus/ContextMenuForContainer.cs

```diff
@@ -68,7 +68,7 @@ protected override IMenuBarItem CreateDeleteItemFor(TContainer objectToRemove)
       private IMenuBarItem createAddExistingFromTemplateItemFor<T>(TContainer parent) where T : class
       {
          var typeName = _objectTypeResolver.TypeFor<T>();
-         return CreateMenuButton.WithCaption(AppConstants.MenuNames.AddExistingFromTemplate(ObjectTypeName))
+         return CreateMenuButton.WithCaption(AppConstants.MenuNames.AddExistingFromTemplate(typeName))
```

Comment:
the wrong object type was used. It should be the child to load (e.g. typename) as opposed to containing object name (container)